client/src/components/Navbar.jsx

```diff
@@ -61,7 +61,8 @@ const Navbar = (props) => {
           {/* Admin auth -> Displays 2 links -> 'Users' and 'Projects'. */}
           {auth?.user?.accessLevel === 'admin' && (
             <>
-              <StyledButton component={NavLink} to="/useradmin">
+              {/* <StyledButton component={NavLink} to="/useradmin"> */}
+              <StyledButton component={NavLink} to="/users/permission-search">
```

Comment:
Based on https://github.com/hackforla/VRMS/issues/1751, the User tab should link to the new "Users" component. 

This change was a part of #1798 PR. So, no need to change anything in this line for this PR.

Reply:
Added comments to clarify that the `UsersPermissionSearch` component from the `"/users/permission-search/"` route is for testing purposes and will be replaced by the new Users component

Reply:
@trillium, can you confirm this is okay, or should this PR include the changes in #1798? 
It should link to "/users". But, for the purpose of testing without the changes in the other PR, it uses "/users/permission-search" instead.